samples/cellular/udp/README.rst

```diff
@@ -136,8 +134,7 @@ Configuration files
 
 The sample provides predefined configuration files for the following development kits:
 
-* :file:`prj.conf` - For nRF9160 DK and Thingy:91
-* :file:`prj_qemu_x86.conf` - For x86 Emulation (QEMU)
+* :file:`prj.conf` - For nRF91 Series and Thingy:91
 
 They are located in :file:`samples/cellular/udp` folder.
```

Comment:
nit: I don't think the configuration files section is needed any more, the prj.conf is standard for all samples.